Watchman.Web/ServiceProviders/AutofacServiceProviderFactory.cs

```diff
@@ -40,5 +44,15 @@ public IServiceProvider CreateServiceProvider(ContainerBuilder containerBuilder)
             container.Resolve<HangfireJobsService>().SetDefaultJobs(container);
             return new AutofacServiceProvider(container);
         }
+
+        private void RegisterCustomServices(ContainerBuilder builder)
+        {
+            var jobs = Assembly.GetAssembly(typeof(AutofacServiceProviderFactory)).GetTypes()
+                .Where(x => x.IsAssignableTo<IhangfireJob>() && !x.IsInterface).ToList();
```

Comment:
po co ToList jak niżej jest sam foreach?

Reply:
albo lepiej `ToList().Foreach(...)`

Reply:
chodzi o łatwość debugowania
jak masz ToList() do zmiennej a dopiero później foreach
to możesz szybko sprawdzić co to zawiera - bez konieczności rozgrzebywania kodu

(chodzi mi o sprawdzenie zawartości całej listy a nie pojedynczego obiektu)